include/bluetooth/mesh/proxy.h

```diff
@@ -21,6 +21,32 @@
 extern "C" {
 #endif
 
+/** Callbacks for the Proxy feature.
+ *
+ *  Should be instantiated with @ref BT_MESH_PROXY_CB_DEFINE.
+ */
+struct bt_mesh_proxy_cb {
+	/** @brief Stopped sending Node Identity beacons on the given subnet.
+	 *
+	 *  @param net_idx Network index the Node Identity beacons were running
+	 *                 on.
+	 */
+	void (*identity_disabled)(uint16_t net_idx);
```

Comment:
I take it we don't want/need `identity_disabled` because this is always explicitly triggered by the application? Is that really true, though, i.e. aren't there some implicit enabling cases like right after provisioning?

Reply:
That is true, it could be triggered automatically. I don't have a usage scenario for this though, which is why I didn't include it. If you think it should be there for consistency, I can add it

Reply:
If it's straight-forward to add I'd just do it, for consistency and since this isn't always explicitly triggered by the application.